### PR TITLE
docs(aio): update headers for toc

### PR DESCRIPTION
**Pull request by Foxandxss**
Verified that there are no weird "entry" in the ToC, removed all the commented out toc (now that I verified that they are nice) and changed the `@title` to h1 (#) continuing Ward's work.

### CODE REVIEW

File: aio/content/guide/webpack.md [hunk 1]
```diff
@@ -781,7 +774,7 @@ Here again are the TypeScript entry-point files that define the `polyfills` and
 
 {@a highlights}
 
-### Highlights
+## Highlights
```
Comment (gkalpak) [inline aio/content/guide/webpack.md]: I am not sure about this, but happy with it if you are.

File: aio/content/guide/router.md [hunk 2]
```diff
@@ -3862,7 +3751,7 @@ drop the `canLoad` guard and rely on the [CanActivate](guide/router#can-activate
 Preloading every lazy loaded modules works well in many situations,
 but it isn't always the right choice, especially on mobile devices and over low bandwidth connections.
 You may choose to preload only certain feature modules, based on user metrics and other business and technical factors.
-
+rm
```
Comment (gkalpak) [inline aio/content/guide/router.md]: ???

File: aio/content/guide/ngmodule-faq.md [hunk 3]
```diff
@@ -550,7 +486,8 @@ from components outside the `AppComponent` tree. This is a rare use case.
 
 More generally, [prefer registering providers in modules](guide/ngmodule-faq#q-component-or-module) to registering in components.
 
-### Discussion
+<h3 class="noToc">Discussion</h3>
```
Comment (gkalpak) [inline aio/content/guide/ngmodule-faq.md]: `noToc` will work, but `no-toc` is what we use in most places :smiley:
That said, why no ToC?

File: aio/content/guide/aot-compiler.md [hunk 4]
```diff
@@ -1,39 +1,9 @@
...
-@intro
-Learn how to use ahead-of-time compilation.
-
-@description
+# Ahead-of-Time Compilation
 
 This cookbook describes how to radically improve performance by compiling _ahead-of-time_ (AOT)
 during a build process.
 
 {@a toc}
```
Comment (gkalpak) [inline aio/content/guide/aot-compiler.md]: Remove?

File: aio/content/guide/attribute-directives.md [hunk 5]
```diff
@@ -136,7 +124,7 @@ Now when the app runs, the `myHighlight` directive highlights the paragraph text
 
 <div class="l-sub-section">
 
-### Your directive isn't working?
+<h3 class="noToc">Your directive isn't working?</h3>
```
Comment (gkalpak) [inline aio/content/guide/attribute-directives.md]: `noToc` will work, but we use `no-toc` in most places :grin:

**Review by gkalpak — CHANGES_REQUESTED** [review]

File: aio/content/guide/router.md (hunk 2 above)
Comment (Foxandxss) [inline aio/content/guide/router.md]: This is called: Multiple monitors and not being sure where your focus is at

File: aio/content/guide/webpack.md (hunk 1 above)
Comment (Foxandxss) [inline aio/content/guide/webpack.md]: In fact, you are right about this one. Not sure what was I thinking about. Or even noToc for this one.

**Comment by Foxandxss** [comment]
Fixed concerns.

**Comment by gkalpak** [comment]
LGTM as soon as Travis is green :grin:

**Comment by petebacondarwin** [comment]
The errors are due to inline tags breaking for some reason. I will look into it

**Review by petebacondarwin — APPROVED** [review]

**Comment by petebacondarwin** [comment]
@Foxandxss  - please squash my commit into yours and then mark as `merge`

**Comment by petebacondarwin** [comment]
@IgorMinar - can you approve the docs change to `common`, please.

**Comment by Foxandxss** [comment]
@petebacondarwin done.

**Review by IgorMinar — APPROVED** [review]

**Review by gkalpak — APPROVED** [review]

**Pull request merged**